src/org/javarosa/core/model/Constants.java

```diff
@@ -42,29 +35,45 @@
     /** Bluetooth connection */
     public static final int CONNECTION_BLUETOOTH = 2;
 
-    /** Data cable connection. Can be USB or Serial */
+    /** Data cable connection. Can be USB or Serial. */
     public static final int CONNECTION_CABLE = 3;
 
     /** Over The Air or HTTP Connection */
     public static final int CONNECTION_OTA = 4;
 
     public static final int DATATYPE_UNSUPPORTED = -1;
-    public static final int DATATYPE_NULL = 0;  /* for nodes that have no data, or data type otherwise unknown */
-    public static final int DATATYPE_TEXT = 1;    /** Text question type. */
-    public static final int DATATYPE_INTEGER = 2;    /** Numeric question type. These are numbers without decimal points*/
-    public static final int DATATYPE_DECIMAL = 3;    /** Decimal question type. These are numbers with decimals */
-    public static final int DATATYPE_DATE = 4;    /** Date question type. This has only date component without time. */
-    public static final int DATATYPE_TIME = 5;    /** Time question type. This has only time element without date*/
-    public static final int DATATYPE_DATE_TIME = 6;    /** Date and Time question type. This has both the date and time components*/
-    public static final int DATATYPE_CHOICE = 7;    /** This is a question with alist of options where not more than one option can be selected at a time. */
-    public static final int DATATYPE_CHOICE_LIST = 8;    /** This is a question with alist of options where more than one option can be selected at a time. */
-    public static final int DATATYPE_BOOLEAN = 9;    /** Question with true and false answers. */
-    public static final int DATATYPE_GEOPOINT = 10; /** Question with location answer. */
-    public static final int DATATYPE_BARCODE = 11; /** Question with barcode string answer. */
-    public static final int DATATYPE_BINARY = 12; /** Question with external binary answer. */
-    public static final int DATATYPE_LONG = 13; /** Question with external binary answer. */
-    public static final int DATATYPE_GEOSHAPE = 14; /** Question with GeoShape answer. */
-    public static final int DATATYPE_GEOTRACE = 15; /** Question with GeoTrace answer. */
+    /* For nodes that have no data, or data type otherwise unknown */
```

Comment:
Does this need to have two * at the beginning?

Reply:
Yes. Thanks!